src/main/java/com/rackspace/ceres/app/validation/IngestBoundsValidator.java

```diff
@@ -0,0 +1,72 @@
+package com.rackspace.ceres.app.validation;
+
+import com.rackspace.ceres.app.config.AppProperties;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.time.*;
+import java.time.temporal.ChronoUnit;
+
+@Log4j2
+@Component
+@Profile("ingest")
+public class IngestBoundsValidator implements ConstraintValidator<IngestBounds, Instant> {
+
+  private final AppProperties appProperties;
+
+  public IngestBoundsValidator(AppProperties appProperties) {
+    this.appProperties = appProperties;
+  }
+
+  @Override
+  public boolean isValid(Instant value, ConstraintValidatorContext context) {
+    log.info("Field Value - {}", value);
+    value = value.atZone(ZoneId.of("UTC")).toInstant();
+    return !(value.isBefore(getStartTime()) || value.isAfter(getEndTime()));
```

Comment:
The logic is a bit backwards, I would prefer !value.isBefore(getStartTime() && !value.isAfter(getStartTime())

Reply:
Fixed it